docs/handbook/systems-administration.md

```diff
@@ -28,109 +28,207 @@ activities and many more are referred to as system administration.
 
 Basic system administration can be reduced to a number of common tasks:
 
+- Users and account management
 - Management of system resources
 - Installation and maintenance of software
-- User administration
 
 While it is certainly possible to add more to this list or select alternative
 items, this small selection is readily absorbed and is convenient to illustrate
 a number of essential concepts central to OpenIndiana system administration.
 
-Before we discuss these topics, it is first important to introduce how these
-tasks can be carried out.
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
+<div class="well">
+Administrative commands are usually expected to be run with elevated privileges -
+directly from root user, via <code>sudo(1M)</code> or
+<code>pfexec</code> (if user was granted privileges via RBAC).
+In this document commands which require elevated privileges are prefixed with
+"# ". Commands, which don't require elevated privileges, are prefixed with "$ ".
+</div>
 
+## Users and account management
 
-## Performing System Administration Tasks
+OpenIndiana allows multiple users to work on the same computer at the same time.
+Only one person can sit in front of the monitor and keyboard.
+However, many can be remotely logged into machine and work on it.
+If a user wants to use the system, he needs an account.
```

Comment:
Modern PC texts prefer to avoid "he" and use "he/she" or "they" (singular in this context). Or can rephrase with a more natural multiple "they", e.g.:
````
If users want to use a system, they need an account.
````

Reply:
A lot of same below.

I did not look if other parts of the document have the issue, maybe it is OK to keep this as is for now and focus on the content, and have separate PR(s) to fix this. And the HTML formatting in the Markdown ;)

Reply:
Maybe here or in the paragraphs a bit below, clarify that many user (and group) accounts are also used for privilege separation of services (and to reduce impact of security breaches) and so are not directly connected to individual human users.

Reply:
As for he/she - this sounds as silly as not calling niggers "niggers"... ;)